src/main/java/edu/ie3/datamodel/models/input/thermal/CylindricalStorageInput.java

```diff
@@ -26,6 +28,37 @@
   /** Specific heat capacity of the storage medium (typically in kWh/K*m³) */
   private final Quantity<SpecificHeatCapacity> c;
 
+  /**
+   * @param uuid Unique identifier of a cylindrical storage
+   * @param id Identifier of the thermal unit
+   * @param operator operator of the asset
+   * @param operationTime operation time of the asset
+   * @param bus Thermal bus, a thermal unit is connected to
+   * @param storageVolumeLvl Available storage volume
+   * @param storageVolumeLvlMin Minimum permissible storage volume
+   * @param inletTemp Temperature of the inlet
+   * @param returnTemp Temperature of the outlet
+   * @param c Specific heat capacity of the storage medium
```

Comment:
Nothing really necessary: But if the common parameters had common definitions and text format throughout...would be great :)